purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -521,18 +684,48 @@ class Purchases internal constructor(
      *
      * @param [activity] Current activity
      * @param [packageToPurchase] The Package you wish to purchase
+     * @param [isPersonalizedPrice] Set this parameter to true if your app can be distributed to users in the EU and
+     * the price you are offering is personalized. Ignored for Amazon.
+     * See https://developer.android.com/google/play/billing/integrate#personalized-price for more info.
      * @param [listener] The listener that will be called when purchase completes.
      */
     fun purchasePackage(
         activity: Activity,
         packageToPurchase: Package,
+        isPersonalizedPrice: Boolean,
         listener: PurchaseCallback
     ) {
         startPurchase(
             activity,
             // TODOBC5 Move this logic to StoreProduct
             packageToPurchase.product.defaultOption?.purchasingData ?: packageToPurchase.product.purchasingData,
             packageToPurchase.offering,
+            isPersonalizedPrice, // TODOC5 figure out isPersonalizedPrice with package
+            listener
+        )
+    }
+
+    /**
+     * Purchase a [Package]. If purchasing a subscription, it will choose the default [SubscriptionOption].
+     *
+     * The default [SubscriptionOption] logic:
+     *   - Filters out offers with "rc-ignore-default-offer" tag
+     *   - Uses [SubscriptionOption] WITH longest free trial or cheapest first phase
+     *   - Falls back to use base plan
+     *
+     * @param [activity] Current activity
+     * @param [packageToPurchase] The Package you wish to purchase
+     * @param [listener] The listener that will be called when purchase completes.
+     */
+    fun purchasePackage(
```

Comment:
Not for this PR but we definitely need to refactor this class… Ideally it should just be a facade to internal classes in charge of each of these functionalities.